### ISSUE TITLE
Error 500 in Logs

### ISSUE THREAD

**Issue by KroMignon** [shinken-monitoring/mod-webui#605]
# Version
Shinken: 2.4.3
WebUI: 2.7.1
OS: Debian GNU/Linux 7.11 (wheezy)

# Problem
Got error 500 when selecting System/Logs

```
Error: 500 Internal Server Error

Sorry, the requested URL 'http://xxxxxxxx:7767/logs' caused an error:

Internal Server Error
Exception:

TypeError("get_ui_logs() got an unexpected keyword argument 'elt'",)
Traceback:

Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 1732, in wrapper
    rv = callback(*a, **ka)
  File "/var/lib/shinken/modules/webui2/module.py", line 509, in lock_version
    return f(**args)
  File "/usr/local/lib/python2.7/dist-packages/bottle.py", line 3615, in wrapper
    result = func(*args, **kwargs)
  File "/var/lib/shinken/modules/webui2/plugins/logs/logs.py", line 185, in get_global_history
    logs = _get_logs(elt=None, filters={'type': {'$in': params['logs_type']}}, range_start=range_start, range_end=range_end)
  File "/var/lib/shinken/modules/webui2/plugins/logs/logs.py", line 45, in _get_logs
    return app.logs_module.get_ui_logs(*args, **kwargs)
  File "/var/lib/shinken/modules/webui2/submodules/logs.py", line 40, in get_ui_logs
    return self.module.get_ui_logs(*args, **kwargs)
TypeError: get_ui_logs() got an unexpected keyword argument 'elt'
```

**Comment by pydubreucq** [comment]
Hi,
Same error for us 
http://forum.shinken-monitoring.org/threads/10630-Impossible-d-acc%E9der-aux-stats-et-logs
Bye

**Comment by maethor** [comment]
Hi,

Thank you for reporting this error. I just fixed it, it will be published in 2.7.3

**Comment by massimiliano-dalcero** [comment]
Hi,
I have the same problem (in 2019 ... two years later).

At the moment the last release of WebUI2 is 2.7.2 and not exists 2.7.3 

there is some news about this error?

Thanks
